### PR TITLE
Add Taylor diags to the webpage

### PR DESCRIPTION
**Pull request by cecilehannay**
This PR adds the Taylor diags to the webpage. There are a few issues with the Taylor diags themselves (I will open an issue). But the issues should be independant from the mechanics of adding the Taylor diags to the webpage.

### CODE REVIEW

**Review by nusbaume — CHANGES_REQUESTED** [review]
Looks good!  I just have one change request.

File: lib/adf_diag.py [hunk 1]
```diff
@@ -1148,9 +1152,13 @@ def create_website(self):
 
                 #Add pressure-level variable to variable list:
                 var_list.extend(pres_var_names)
+
             #End for
         #End if
 
+        # add Taylor diagrams plot (Cecile needs to add)
+        var_list.extend(['cam'])
```
[the remark this answers is not in the document]
Comment (cecilehannay) [reply lib/adf_diag.py]: Good suggestion. I did the change.

**Review by nusbaume — APPROVED** [review]
Looks great now, thanks!

**Comment by nusbaume** [comment]
@justin-richling just pinging you to see  if you could move the "missing webpage" fix to the top of your to-do list now.  Thanks!

**Comment by justin-richling** [comment]
@nusbaume Thanks, I'm on it now. I should have a PR coming your way soon!

**Pull request merged**